data/videos.yml

```diff
@@ -0,0 +1,35 @@
+- title: What is time to awesome?
+  id: iMKCe3WKFo0
+  description: Placeholder 1
+  pagename: what-is-time-to-awesome
+  tags: []
+
+- title: Placeholder 2
+  id: iMKCe3WKFo0 
+  description: Placeholder 2
+  pagename: test2
+  tags: []
+
+- title: Placeholder 3
+  id: VJvinU8_f00
+  description: Placeholder 3
+  pagename: test3
+  tags: []
+
+- title: Placeholder 4
+  id: Ymp28MxosFQ
+  description: Placeholder 4
+  pagename: test4
+  tags: []
+
+- title: Placeholder 5
+  id: EzFEy-xJYhM
+  description: Placeholder 5
+  pagename: test6
+  tags: []
+
+- title: Placeholder 6
+  id: Mj1KdzC6EKU
+  description: Placeholder 6
+  pagename: test6
+  tags: []
```

Comment:
curious about the test placeholders--do we want to update pagenames?

Reply:
Oh, we don’t need this data file. I forgot to remove it.